website/docs/related-tools.md

```diff
@@ -32,3 +32,7 @@ A [GitLab Pipeline](https://github.com/oss-review-toolkit/ort-gitlab-ci) to run
 ## ORTHW
 
 A [bash script](https://github.com/oss-review-toolkit/orthw) that helps to simplify and speed up common tasks performed when processing ORT results.
+
+## OpossumUI
+
+[OpossumUI](https://github.com/opossum-tool/OpossumUI) is a desktop application that provides a compliance audit workflow and can load ORT result files.
```

Comment:
Maybe say "import" or "handle" instead of "load" ?

Reply:
I have replaced it by mentioning the "Opossum" reporter.